var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -70,6 +70,11 @@ class Fftw(AutotoolsPackage):
     depends_on('autoconf', type='build', when='+pfft_patches')
     depends_on('libtool', type='build', when='+pfft_patches')
 
+    @property
+    def libs(self):
+        result = find_libraries(['*fftw3'], root=self.prefix, recurse=True)
```

Comment:
If I just write `fftw3`, it doesn't find `libfftw3.so` since `find_libraries` doesn't add the `lib` prefix ([see here](https://github.com/LLNL/spack/blob/9e8cce41e3f78ed4222eff342a5b61b6f8fbf645/lib/spack/llnl/util/filesystem.py#L971)). It just looks for `fftw.so` and fails to find it. I think using `libfftw3` instead would be better, since I checked the other uses of `find_libraries` and this is what they seem to do.